index.html

```diff
@@ -6,231 +6,254 @@
+
+    <p>
+    1. "Placed documents" - Documents, more or less permanent, that are accessible to one or more users; these may be set to allow further editing by a subset of those users.
+    </p>
+
+    <p>
+    2. "Input Flows" - Documents created by a user to be "pushed through" down the line from user to user, with each given the opportunity to provide input before pushing it to the next user until it reaches the end of the line. At this point, it may either be converted into a placed document or sent back up the line for further input; this process may be repeated as many times as desired by the flow&#39;s originator. This may be used for such things as brainstorming and assessing news items and other data. That each user is directly confronted with the document, which remains in his module until being passed, helps to focus participants and encourage contributions (an option may also be set that automatically pushes the document through after some particular amount of time has passed without editing, in which case a notation is appended to that effect).
+    </p>
+
+    <p>
+    3. "Submission Flows" - Documents created by a user to be pushed up the line, generally for the purpose of submitting ideas, proposals, and items of information, which each user may either push forward or not as deemed appropriate. This constitutes one of the basic mechanics of the pursuance system as "filter," and indeed was one of its original purposes; a pursuance that starts out with a few high-quality users can grow to include hundreds without any real risk of becoming cluttered with low-quality input coming from the margins, as items must be approved forward by each participant up the line.
+    </p>
+
+    <p>
+    As with several other aspects of the system, the flow process is subject to exceptions that may be determined via settings. By default, any participant may send one document to the primary or anyone else up the line at any time, outside of structural channels; if the document turns out to be worthwhile, the recipient may choose to renew the sender's ability to do this. Another setting determines whether a submission flow document, upon being created and sent up the line, also appears randomly at some other point in the pursuance, or at several points, where they may be pushed up the line as usual; this is intended to speed up the proliferation of time-sensitive news items, as well as to better ensure that good input from the margins get more than one chance for acceptance. Another setting allows for documents to automatically proceed to the next position on the line after some variable amount of time has passed without editing or approval/disapproval; this is largely intended to prevent bottlenecking due to temporary absences. Other settings will allow for additional refinement within a pursuance's existing structure.
+    </p>
+
+    <a class="btn btn-primary nav-link" href="/donate.html">Donate to Pursuance!</a>
+    <br />
+    <br />
+  </div>
```

Comment:
@seabassgonzalez Please don't change the indentation level if we're not also editing the text; someone else will soon create a PR where the "Broader Vision" section is changed and avoiding merge conflicts based on mere whitespace differences would be great!

Reply:
Got it